### DISCUSSION TITLE
Change landmark sign for different religious place

### DISCUSSION THREAD

**Discussion by anastasiia936** [osmandapp/OsmAnd#12352] [Ideas]
Different religion has different way of worshipping. But in OsmAnd all religions show with the same sign  (a man praying in a position preferred in Islam). It would be great if OsmAnd adds different signs for different religious places. 
![image](https://user-images.githubusercontent.com/80623554/127861671-a5941433-6a34-4266-b5a3-5799f636059d.png)
![image](https://user-images.githubusercontent.com/80623554/127861686-c1a10b64-5bdd-46d7-9856-6e2a5f3fc1aa.png)

**Comment by tommycrock** [comment]
I've seen plenty of Christians in that pose while praying, and searching for "[insert religion] praying kneeling" brings up similar images for a range of religions. The kind of religion is also shown on the map, with a shared symbol for place of worship. It seems like a pose that denotes worship even if not practiced in a particular religion.
